tests/api/test_web_auth_hardening.py

```
@@ -349,3 +549,216 @@ async def boom():
+    app.add_middleware(AuthenticationMiddleware, config=AuthConfig())
+    app.include_router(router)
+    return app
+
+
+@pytest.fixture
+def local_auth_sse_app(monkeypatch: pytest.MonkeyPatch) -> FastAPI:
+    from penguin.web.middleware.auth import AuthenticationMiddleware
+
+    monkeypatch.setenv("PENGUIN_AUTH_ENABLED", "true")
+    event_bus = SimpleNamespace(
+        subscribe=lambda *args, **kwargs: None,
```

Ruff (0.15.10)

[warning] Unused lambda argument: `args`

(ARG005)